pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "armasec"
-version = "2.0.1"
+version = "2.0.2"
```

Comment:
I think this is a breaking change, specially because all the testing code for software that use armasec will need to be reviewed

Reply:
actually, can we launch an alpha version with this change so we can run some tests?

Reply:
disregard my comment